pil-analyzer/src/type_builtins.rs

```diff
@@ -48,6 +48,7 @@ lazy_static! {
         ("std::field::modulus", ("", "-> int")),
         ("std::prelude::challenge", ("", "int, int -> expr")),
         ("std::prover::degree", ("", "-> int")),
+        ("std::prover::add_hint", ("", "expr, (int -> fe) ->")),
```

Comment:
Should we call it `add_hint` or `set_hint`? We probably won't support more than one hint...

Reply:
`set_hint` makes sense then